repository-data/webfiles/src/main/resources/site/freemarker/intranet/home-static.ftl

```diff
@@ -0,0 +1,402 @@
+                            </div>
+                        </article>
+                    </li>
+                    <li>
+                        <article class="intra-articleblock">
+                            <h1 class="intra-articleblock__title"><a href="#">KOSH Training</a></h1>
+                            <div class="intra-articleblock__text rich-text-content">
+                                <p>Training regulations and schedule has now changed. Please check the schedule for more information </p>
+                            </div>
+                        </article>
+                    </li>
+                </ul>
+
+                <@actionLink title="View all Articles" link="#" />
+            </div>
+
+            <div class="intra-box intra-box--nested">
+                <h2 class="intra-box__title">NHS Digital Jobs</h2>
+
+                <ul class="intra-articleblock-group">
+                    <li>
+                        <article class="intra-articleblock">
+                            <h1 class="intra-articleblock__title"><a href="#">Communications Officer</a></h1>
+                            <span class="intra-articleblock__subtitle">Leeds</span>
+                            <div class="intra-articleblock__text rich-text-content">
+                                <p><b>Salary:</b> £40,729.91</p>
+                                <p><b>Posted:</b> 28/01/2020</p>
+                                <p><b>Job Type:</b> Permanent</p>
+                            </div>
+                        </article>
+                    </li>
+                    <li>
+                        <article class="intra-articleblock">
+                            <h1 class="intra-articleblock__title"><a href="#">Higher Information Analyst</a></h1>
+                            <span class="intra-articleblock__subtitle">Leeds</span>
+                            <div class="intra-articleblock__text rich-text-content">
+                                <p><b>Salary:</b> £40,729.91</p>
+                                <p><b>Posted:</b> 28/01/2020</p>
+                                <p><b>Job Type:</b> Permanent</p>
+                            </div>
+                        </article>
+                    </li>
+                    <li>
+                        <article class="intra-articleblock">
+                            <h1 class="intra-articleblock__title"><a href="#">User Researcher</a></h1>
+                            <span class="intra-articleblock__subtitle">Leeds</span>
+                            <div class="intra-articleblock__text rich-text-content">
+                                <p><b>Salary:</b> £40,729.91</p>
+                                <p><b>Posted:</b> 28/01/2020</p>
+                                <p><b>Job Type:</b> Permanent</p>
+                            </div>
+                        </article>
+                    </li>
+                </ul>
+
+                <@actionLink title="View all Jobs" link="#" />
```

Comment:
What is this file for?

Reply:
It was the static file created by the front-end for the intranet, i didn't want to get rid of it as it may be required for future developments. Added it to the site-development repository data which is not bootstrapped in production (only local env).